spec/features/user/retired_item_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature 'when an item is retired' do
+  let(:category) { Category.create!(name: "Gross")}
+
+  before(:each) do
+    @item = Item.create(title: "Eat Cheese and Die",
+                        description: "The spookiest grilled cheese.",
+                        price: 50.65,
+                        categories: [category],
+                        retired: true)
+    @user = User.new(email: "what@whatwhat.com",
+                     password: "Seekrit",
+                     full_name: "Sir Whats",
+                     display_name: "mrwhatsalot")
+    
+    allow_any_instance_of(ApplicationController).to receive(:current_user) { @user }
+  end
+  
+  scenario 'user can still visit item page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.